.github/workflows/signoff-check.yml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A workflow to check if PR got sign-off
+name: signoff check
```

Comment:
I think moving the signoff check to Github actions would be better handled as a separate PR rather than lumping it in here.  It doesn't seem related to moving to Blossom.

Reply:
yep, this will be a separate PR. Just left here to give people a full picture :)